outlook/src/taskpane/components/Log/Logger.tsx

```diff
@@ -6,6 +6,13 @@ import {Spinner, SpinnerSize, TooltipHost} from "office-ui-fabric-react";
 import {FontAwesomeIcon} from "@fortawesome/react-fontawesome";
 import {faCheck, faEnvelope} from "@fortawesome/free-solid-svg-icons";
 import {OdooTheme} from "../../../utils/Themes";
+import {_t} from "../../../utils/Translator";
+
+//total attachments size threshold in MB
+const totalSizeThreshold = 40;
+
+//single attachment size threshold in MB
+const elementSizeThreshold = 10;
```

Comment:
Real constants are usually uppercased and snake_cased.

So it would be:
```js
//total attachments size threshold in MB
const SIZE_THRESHOLD_TOTAL = 40;

//single attachment size threshold in MB
const SIZE_THRESHOLD_SINGLE_ELEMENT = 10;
```

(Also better when similar things have the same prefix, here "size_threshold_XXX").

Reply:
Constant names changed!